src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/ExpressionBinder.cs

```diff
@@ -1560,12 +1560,12 @@ private Expr AdjustMemberObject(SymWithType swt, Expr pObject, out bool pfConstr
 
             if (typeObj is TypeParameterType || typeObj is AggregateType)
             {
-                AggregateSymbol aggCalled = swt.Sym.parent.AsAggregateSymbol();
+                AggregateSymbol aggCalled = swt.Sym.parent as AggregateSymbol;
```

Comment:
@OmarTawfik this line was the only conflict on rebasing since the changeset you already reviewed. Just too close to the change 2 lines above for the merge tool, I suppose.